src/openms/source/ANALYSIS/MRM/ReactionMonitoringTransition.cpp

```diff
@@ -284,4 +284,36 @@ namespace OpenMS
     library_intensity_ = intensity;
   }
 
+  bool ReactionMonitoringTransition::isDetectingTransition() const
+  {
+    bool detecting = true;
+    if (this->getMetaValue("detecting_transition").toString() == "0") {detecting = false;}
```

Comment:
Should we fix this directly in DataValue? What would be the preferred solution? Storing boolean DataValue as xsd:string in the XML files or making toBool() compatible with the currently stored integers?